### ISSUE TITLE
hi

### ISSUE THREAD

**Issue by wangweifei-504** [JHyunjun/TF2.0_AutoEncoder-with-LSTM#1]
Hello, can you explain the function of your code? And can the dataset be shared?

**Comment by JHyunjun** [comment]
Hello Wangweifei

I used time-series double-type data for anomaly detecting using AutoEncoder.
I can't share about it but that type of data is commonly available in
Kaggle or etc..(ex. Well known : house-price dataset in Kaggle)

overall that model used same feature/dimension in/output data for training
(Auto Encoding)

*Input layer ->* model.add(layers.LSTM(units = hiddenDim,
activation='tanh',input_shape = (seqLength,*input_dim*)

*output layer ->* model.add(layers.Dense(*input_dim*, activation='linear'))


and, LSTM layer was used for Encoding-Decoding

model.add(layers.LSTM(units = hiddenDim, activation =
'tanh',return_sequences = True, kernel_initializer =
initializer))model.add(layers.LSTM(units = hiddenDim, activation =
'tanh',return_sequences = False, kernel_initializer = initializer))

I think above code are the most important part of my code

If you have any question about other part of the code or etc, whenever
send me msg pls :)


2022년 1월 9일 (일) 오후 3:50, wangweifei-504 ***@***.***>님이 작성:

**Comment by wangweifei-504** [comment]
Hi sir,thank you for your reply. I would like to know do you use the reconstruction error to judge the abnormal data?

**Comment by JHyunjun** [comment]
Oh, i didn't use reconstruction error for judging.
I use just Loss value between Input and output data :)
I reconstructed all of my data, but that's not for judging, just for
visualizing

2022년 1월 10일 (월) 오전 11:45, wangweifei-504 ***@***.***>님이 작성:

**Comment by wangweifei-504** [comment]
Is it your job to make the output well reconstruct the input? Or has your paper been published and can you share it?

**Comment by JHyunjun** [comment]
yes, well reconstructed output is very important to me for writing paper.
but It has nothing to do with the operation of the autoencoder. I just did
it for visualizing and graph
i wrote and registered patent and paper recently, so there weren't
published now :(



2022년 1월 11일 (화) 오후 10:26, wangweifei-504 ***@***.***>님이 작성:

**Comment by wangweifei-504** [comment]
Ok, thank you for your reply, I wish your paper can be published smoothly.
